code/SNIDsn.py

```diff
@@ -463,7 +464,59 @@ def loadSNIDlnw(self, lnwfile):
             continuum[ind - 1] = np.array([float(x) for x in cont_line])
         self.continuum = continuum
         return
+    
+    def loadSN(self, file, phaseType):
+        """
+        Loads an ascii SNID template file specified by the path file into
+        a SNIDsn object. The file must only have wavelength in the first column
+        and fluxes in subsequent columns. The header line must also have the phases,
+        with the first entry in the header line being the phase type.
+
+        Parameters
+        ----------
+        file : string
+            path to SNID template file produced by logwave.
+        phaseType : integer
+            value that specifies whether phases are defined relative to max
 
+        Returns
+        -------
+
+        """
+        
+        assert file[-3:] != 'lnw' #file cannot be .lnw
+        self.phaseType = phaseType
+        
+        all_data = np.loadtxt(file, skiprows = 1)
+        
+        header_line = all_data[0]
+        phases = header_line[1:len(header_line)]
+        self.phases = phases
+        
+        wvl = []
+        for row in all_data:
+            wvl.append(row[0])
+        wvl.pop(0)
+        wvl = np.array(wvl)
+        self.wavelengths = wvl
```

Comment:
np.loadtxt already gives you an ndarray with array slicing ability.

`self.wavelengths = all_data[:, 0]` grabs entries in every row of the 0th column.

Reply:
`self.wavelengths = all_data[1:, 0]` skips the first row

Reply:
`self.wavelengths = all_data[[1,2,3,4], 0]` which gives you the wavelengths in rows 1,2,3,4